### PR TITLE
Add coordinates for remarqued statistic points

### PR DESCRIPTION
**Pull request by nono303**
https://github.com/Sibyx/phpGPX/issues/63

### CODE REVIEW

**Comment by Sibyx** [comment]
@nono303 can you please have a look?

**Comment by nono303** [comment]
Hi @Sibyx 
Can't see why Merging is blocked ( as i only add new properties. and didn't have permission to approve.
only think I view is a "false" deletion in /src/phpGPX/Models/Stats.php l.95<>119.
I'll try reorder changes to see if it allow merging

**Review by nono303 — COMMENTED** [review]
No breaking changes

**Comment by nono303** [comment]
> didn't have permission to approve  

image

**Comment by Sibyx** [comment]
@nono303 I am the only one who can approve the MR. But please respond to my question in the code review. Thank you :)

**Comment by nono303** [comment]
hmmmm... that's why I didn't understand 'cause I didn't have any question to answer in code review :)
image
So what's your question ?

**Review by Sibyx — CHANGES_REQUESTED** [review]

**Comment by Sibyx** [comment]
I'm so sorry, @nono303. I forgot to submit the review and it was pending on my side all this time.

File: src/phpGPX/Models/Route.php [hunk 1]
```diff
@@ -109,14 +112,17 @@ public function recalculateStats()
...
 			}
 
 			if ($this->stats->maxAltitude < $this->points[$p]->elevation) {
 				$this->stats->maxAltitude = $this->points[$p]->elevation;
+				$this->stats->maxAltitudeCoords = ["lat" => $this->points[$i]->latitude, "lng" => $this->points[$i]->longitude];
 			}
 
 			if ($this->stats->minAltitude > $this->points[$p]->elevation) {
 				$this->stats->minAltitude = $this->points[$p]->elevation;
+				$this->stats->minAltitudeCoords = ["lat" => $this->points[$i]->latitude, "lng" => $this->points[$i]->longitude];
```
[the remark this answers is not in the document]
Comment (nono303) [reply src/phpGPX/Models/Route.php]: hi @Sibyx, you're right, it's $p !
fixed in https://github.com/Sibyx/phpGPX/pull/64/commits/5abd3c33aa15ec746836e6e5e15a26f72b8a4d56

**Review by Sibyx — APPROVED** [review]
Looks good. May i ask you to mention it somewhere in the docs or README? I know the current state of the docs is not that good. I hope I will resolved someday in the long planned 2.x release (feel free to help if you have time for that).

**Comment by nono303** [comment]
> Looks good @nono303. May i ask you to mention it somewhere in the docs or README? I know the current state of the docs is not that good. I hope I will resolved someday in the long planned 2.x release (feel free to help if you have time for that).

I'll do it next week

**Comment by Sibyx** [comment]
> > Looks good @nono303. May i ask you to mention it somewhere in the docs or README? I know the current state of the docs is not that good. I hope I will resolved someday in the long planned 2.x release (feel free to help if you have time for that).
> 
> I'll do it next week

Sure, take your time as you need. Thank you for your contribution :)

**Comment by nono303** [comment]
Hi @Sibyx ,
Let you review [063adf1](https://github.com/Sibyx/phpGPX/pull/64/commits/063adf12f6baddda577fdb479bfcc9585361fdd5) and tag it as 1.2.2 if OK for you

**Comment by Sibyx** [comment]
Love it, thank you very much for your contribution!

**Pull request merged**